console/application.go

```diff
@@ -8,21 +8,26 @@ import (
 
 	"github.com/goravel/framework/contracts/console"
 	"github.com/goravel/framework/contracts/console/command"
-	"github.com/goravel/framework/support"
 	"github.com/goravel/framework/support/color"
 )
 
 type Application struct {
-	instance *cli.App
+	instance  *cli.App
+	isArtisan bool
 }
 
-func NewApplication() console.Artisan {
+func NewApplication(name, usage, usageText, version string, artisan ...bool) console.Artisan {
 	instance := cli.NewApp()
-	instance.Name = "Goravel Framework"
-	instance.Usage = support.Version
-	instance.UsageText = "artisan [global options] command [options] [arguments...]"
-
-	return &Application{instance}
+	instance.Name = name
+	instance.Usage = usage
+	instance.UsageText = usageText
+	instance.Version = version
```

Comment:
What will be added in the terminal after adding this field?

Reply:
It will add one more flag: --version(-v). We don't have to implement it ourselves. 
### Without version
![image](https://github.com/goravel/framework/assets/84431594/f3a39ecd-0c6f-4119-a5a2-f163bbc5082c)
### With version
![image](https://github.com/goravel/framework/assets/84431594/72e3d2ba-2119-41a2-997f-6c9dd73a2bc0)
![image](https://github.com/goravel/framework/assets/84431594/d88abd3b-ddb4-4de9-bbd9-afd9835ea669)